sdf/embedSdf.py

```diff
@@ -61,60 +83,39 @@ def get_file_header_prolog() -> str:
     const std::map<std::string, std::string> &GetEmbeddedSdf()
     {
         static const std::map<std::string, std::string> result {
-    """)
+    """
+    )
     return res + NEWLINE
 
 
 def embed_sdf_content(arg_path: str, arg_file_content: str) -> str:
     """
-    Generates a string pair with the folder and filename as well as the content of the file
+    Generates a string pair with the folder and filename
+    as well as the content of the file
 
     :param arg_path: Foldername and filename of the SDF
     :param arg_file_content: Content of the provided file
     :returns: raw string literal mapping pair for the std::map
     """
     res = []
-    res.append('// NOLINT')
-    res.append('{')
+    res.append("// NOLINT")
+    res.append("{")
     res.append(f'"{arg_path}",')
     res.append('R"__sdf_literal__(')
-    res.append(f'{arg_file_content}')
+    res.append(f"{arg_file_content}")
     res.append(')__sdf_literal__"')
-    res.append('},')
-    res.append('')
+    res.append("}")
     return NEWLINE.join(res)
 
 
-def get_map_content(arg_pathlist: Path) -> str:
-    """
-    Generates a string pair with the folder and filename as well as the content
-    of the file in ascending order
-
-    :param arg_pathlist: Foldername and all filenames inside it
-    :returns: mapping pairs for the std::map
-    """
-    map_str = ''
-    files = []
-    for path in arg_pathlist:
-        # dir separator is hardcoded to '/' in C++ mapping
-        posix_path = PurePosixPath(path)
```

Comment:
we've seen regressions on windows when porting this forward in https://github.com/gazebosim/sdformat/pull/1297#pullrequestreview-1515116022

I think we need to keep the `PurePosixPath` call